zerver/lib/actions.py

```diff
@@ -5008,6 +5008,17 @@ def do_create_realm(
             realm.demo_organization_scheduled_deletion_date = (
                 realm.date_created + datetime.timedelta(days=settings.DEMO_ORG_DEADLINE_DAYS)
             )
+
+        # Changing default permission policy for education organisations
+        if org_type == 30 or org_type == 35:
```

Comment:
Please use the named constants defined in `models.py`, rather than their integer values -- the code is very hard to read otherwise :).

You should also add tests that verify this logic is working correctly.

Reply:
@timabbott I'll replace the int values with the defined constants in `models.py`. I was having doubts about adding tests like what files I have to add tests to, I think it should be added to `zerver/tests/test_realm.py`. I really want to learn about testing.

Reply:
https://zulip.readthedocs.io/en/latest/testing/testing-with-django.html is helpful background.  I think tests for this feature could reasonably go in `test_realm.py` or `test_signup.py` (if one wanted to do more of an end-to-end signup test).

Reply:
@madrix01: yes that file should be fine! Usually, if you look at the existing tests in the file, it can help one understand how to write one for your purposes. :)

Reply:
@eeshangarg @timabbott  I'll try to read the documentation and create some test cases for the logic I have implemented.